Makefile

```diff
@@ -0,0 +1,33 @@
+.POSIX:
+SHELL=/bin/sh
+BASE_PATH="/"
+
+.PHONY: ci-test
+ci-test: 
+	@echo "Install dependencies" 
+	npm ci
+	@echo "Check code formatting"
+	npm run check-format
+	@echo "Lint code"
+	npm run lint
+	@echo "Generate production build"
+	npm run export
+
+.PHONY: ci-deploy-prod
+ci-deploy-prod:
+	@echo "Generate production build"
+	npm run export -- --basepath=$(BASE_PATH)
+	if ! [ -x "$$(command -v aws)" ]; then\
+		@echo "Install AWS CLI";\
+		curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "$(HOME)/awscliv2.zip";\
+		unzip $(HOME)/awscliv2.zip -d $(HOME);\
+		sudo $(HOME)/aws/install;\
+	fi
+	@echo "Configure AWS Credentials"
+	aws configure set aws_access_key_id $(AWS_ACCESS_KEY_ID)
```

Comment:
Can you confirm whether AWS CLI can read from environment variable directly? Configuring it like this would change user's default profile.

Reply:
you're right, AWS CLI can read env variables directly